python/lsst/ap/pipe/ap_pipe.py

```diff
@@ -99,9 +99,17 @@ def setDefaults(self):
         self.differencer.doSelectSources = False
 
         self.associator.level1_db.retarget(AssociationDBSqliteTask)
+        # TODO: generalize in DM-12315
+        self.associator.level1_db.filter_names = ['u', 'g', 'r', 'i', 'z', 'y', 'VR', 'N964']
```

Comment:
Why doesn't using a configfile for in the CmdLineTask work here? This looks very similar to how it would happen in such a file. I'm worried that this may be adding in values specific to DECam.

Reply:
Almost everything in the current default config is DECam-specific; DM-12315 will fix that (see also the Jira description of DM-14497).

Reply:
Sounds good.